reference/ps-modules/Microsoft.PowerShell.SecretManagement/Set-SecretVaultDefault.md

```diff
@@ -66,19 +66,18 @@ CredMan    Microsoft.PowerShell.CredManStore False
 LocalStore Microsoft.PowerShell.SecretStore  False
 ```
 
-This cmdlet first runs `Get-SecretVault` command to get all registered vault information, and shows
-that the 'LocalStore' is currently the default vault for the user. Next, the
-`Set-SecretVaultDefault` command is run to make the 'CredMan' vault the default vault. The
-`Get-SecretVault` command is run a second time to verify 'CredMan' vault is now default, and
-'LocalStore' vault is no longer default. Finally, the `Set-SecretVaultDefault` command is run with
-no **Name** parameter, to remove the default designation from any registered vault. The
-`Get-SecretVault` is run once again to verify there is no default vault.
+This example shows how to set and unset the default vault for the current user. The first command
+gets information about all registered vaults and shows that the `LocalStore` is the default vault
+for the user. The second command makes the `CredMan` vault the default vault. The third command
+shows that the `CredMan` vault is now default, and `LocalStore` vault is no longer default. The
+fourth command uses the **ClearDefault** parameter to remove the default designation from any
+registered vault. The last command shows that there is no default vault.
```

Comment:
This description is all wrong.

Reply:
Looking it over again, that's because the PR preview cut out the extent of the example, which is:

```powershell
PS C:\> Get-SecretVault
VaultName  ModuleName                        IsDefaultVault
---------  ----------                        --------------
CredMan    Microsoft.PowerShell.CredManStore False
LocalStore Microsoft.PowerShell.SecretStore  True
PS C:\> Set-SecretVaultDefault -Name CredMan
PS C:\> Get-SecretVault
VaultName  ModuleName                        IsDefaultVault
---------  ----------                        --------------
CredMan    Microsoft.PowerShell.CredManStore True
LocalStore Microsoft.PowerShell.SecretStore  False
Set-SecretVaultDefault -ClearDefault
Get-SecretVault
VaultName  ModuleName                        IsDefaultVault
---------  ----------                        --------------
CredMan    Microsoft.PowerShell.CredManStore False
LocalStore Microsoft.PowerShell.SecretStore  False
```

Reply:
I would break it into 2 examples. One to change the default. One to remove all defaults.